src/main/java/com/example/PascalTriangle.java

```diff
@@ -1,11 +1,49 @@
 package com.example;
 
 public class  PascalTriangle{
+	public static int factorial(int number) {
+		int fact = 1;
+		for(int i = 1; i <= number; ++i) {
+			fact *= i;
+		}
+		return fact;
+  }
+
+  /*
```

Comment:
Don't leave commented-out code in the code base. Git history will show what was before. Even people looking at the pull request at a later time will see what was the discussion, what's changed and why.

Reply:
OK.